vignettes/announcement_addin.Rmd

```diff
@@ -0,0 +1,129 @@
+- **Input calendar default value**: current day
+- Date Rules
+  - Either of Start/End dates inputs or both can be NULL
+  - Start Date calendar minimum date will always be current day (use won't be able to select earlier days)
+  - End Date calendar minimum date will be either
+    - Current day if start date  is NULL
+    - or start date day if it is selected as below
+    
+    ![End Date Calendar Minimum Value](figures/add_ins_announce_4.png){#fig-4}
+    
+  - If user selected an end date before selecting start date, then select a start date that is later than end date, end date will be updated to the selected start date
+  
+![](figures/add_ins_announce_5.png){#fig-5}
+
+![](figures/add_ins_announce_6.png){#fig-6}
+
+### Close after (sec)
+
+ - **Function**: Numeric input to determine how long announcement should be displayed
+- * *Default value**: 30
+- **Tooltip**: "Time, in seconds, to auto close announcement banner after that time elapsed. Leave value blank or zero to leave announcement bar open until user closes it manually."
+Input Rules:
+Value can be 0 or NULL
+It cannot be characters
+It cannot be Negative
+
+![Close after](figures/add_ins_announce_7.png){#fig-7}
+
+
+### Style
+
+- **Function**: Dropdown to select announcement alert color
+- **Default value**: primary
+- **Tooltip**: "Color for the announcement banner, possible values are {'primary', 'success', 'warning', 'danger' or 'info'}. It is a mandatory value."
+- **Input Rules**:
+  - It cannot be NULL
+  - It cannot have a different value than what is listed in the dropdown
+  
+![Style](figures/add_ins_announce_8.png){#fig-8}
+
+
+### Title
+- **Function**: Set announcement title 
+- **Default value**: empty
+- **Tooltip**: "Optional banner title. Leave it empty to disable it"
+- **Input Rules**: It is not mandatory
+- **Placeholder**: "Announcement Banner Title"
+
+![Title](figures/add_ins_announce_9.png){#fig-9}
+
+
+### Announcement Text
+
+- **Function**: Set announcement body
+- **Default value**: empty
+- **Tooltip**: "The announcement text. Text can contain html tags and is a mandatory value"
+- **Input Rules**: It is mandatory
+- **Placeholder**: "Announcement Banner Title"
+- Its size can be increased only by dragging the bottom left corner
+
+![Announcement Text](figures/add_ins_announce_10.png){#fig-10}
+
+![Announcement Validation](figures/add_ins_announce_11.png){#fig-11}
+
+### Download button
+
+- **Function**: Download announcement configuration yaml file
+- **Default State**: Disabled
+- **Rules**: It will be enabled if all mandatory inputs are provided and valid
+- **Label**: "Download"
+
+### Downloaded File
+
+- **Name**: announce.yaml
+- **Format**: Value are based on input configuration but output is similar to the following
+![Announcement File](figures/add_ins_announce_12.png){#fig-12}
```

Comment:
I think the vignette is missing where to put the downloaded announcement file afterwards to use it. I think that will help the users know what to do after creating the file

Reply:
I think telling them which folder to put it in their app created by periscope2 will be enough

Reply:
The details of that is in another vignette, I pointed to it here. Done